modules/proc-providing-feedback-on-redhat-documentation.adoc

```diff
@@ -0,0 +1,46 @@
+[preface]
+[id="proc-providing-feedback-on-redhat-documentation"]
+= Providing feedback on Red Hat documentation
+
+We appreciate your feedback on our documentation.
+To provide feedback, you can highlight the text in a document and add comments.
+
+This section explains how to submit feedback.
+
+.Prerequisites
```

Comment:
IMO this shouldn't really be a procedural topic with distinct `.Prerequisites` and `.Procedure` sections. I guess we can go with it but it seems kind of weird. I feel like this should be a lot lighter and more conversational. I mean, we are documenting a UI here, something that users can discover themselves. I just think it's a little carried away to make it more of a formal procedure.

Reply:
I'm okay with it not being a procedure, but I didn't want a paragraph explaining how to leave feedback, so the numbered list made sense.  I'm open to suggestions.

PS. This was copied from RHEL and edited to meet Runtimes' needs.

Reply:
Right. I think we should have an ordered list for the procedure too (not a paragraph). I just don't think this topic really needs to follow the modular template, which seems a little overkill.